spec/rubocop/cop/rspec/context_wording_spec.rb

```diff
@@ -214,4 +214,17 @@
       end
     end
   end
+
+  context 'when `AllowedPatterns:` and `Prefixes:` are both empty' do
```

Comment:
memo) In this case, it is pointless to enable this cop because it is not always an offense. Should we occur warning or runtime error for this setting? WDYT? @rubocop/rubocop-rspec

Reply:
>  it is not always an offense

I don’t understand. Do you mean it’s not deterministic?

Reply:
The following checks are always considered non-offenses:

https://github.com/rubocop/rubocop-rspec/blob/7592cecc0a16e2f73475e5b60c72d0baa13b2a29/lib/rubocop/cop/rspec/context_wording.rb#L88

This Cop sets valid wording in Prefixes and AllowedPatterns, but it does not always consider a offenses if there is no valid wording setting.

Reply:
Right, with these settings, there will be no offenses, regardless of what is or is not in the Context. It is always not an offense. (I'm not attached to it staying that way). 

I looked at the Obsoletion setup recently, and I think it raises runtime errors with the expectation that the user will want to fix those issues. That feels similar to how a user may want to fix the config or disable this cop instead of letting it run "on empty"

Reply:
It’s an unlikely combination of settings, so unless it’s very simple to implement, I don‘t think we need to warn our users about it.

Reply:
Maybe if the `Prefixes` and `AllowedPatterns` are empty to begin with, there are no allowed Wording, so maybe it's better to make them all violations. That way, users would be able to notice any misconfigurations and the implementation wouldn't be that hard.